<a href="https://colab.research.google.com/github/Deniska122334/A-B-testing-H_W/blob/main/H_W_5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## Задание 1

In [1]:
import numpy as np
import statsmodels.stats.proportion as smp

In [2]:
alpha = 0.05
# число успехов в каждой группе
successes = np.array([164, 228])

# число наблюдений в каждой группе
nobs = np.array([15550, 15550])

# пропорции успехов в каждой группе
proportions = successes / nobs

# z-статистика и p-значение
zstat, pvalue = smp.proportions_ztest(count=successes, nobs=nobs, value=0, alternative='two-sided')
print(zstat)
print(pvalue)

-3.253054722696254
0.001141715077067056


На основе z-статистики: -3.253054722696254 и p-value: 0.001141715077067056, можно сделать вывод о статистически значимых различиях между конверсиями в двух группах.

** Рекомендации команде: на основании результатов эксперимента можно рекомендовать использовать упрощенную форму заказа в магазине Утконос, так как она приводит к более высокой конверсии в покупку. Однако, следует провести дополнительные исследования, чтобы понять, какие именно элементы формы заказа необходимо упростить, чтобы максимизировать конверсии.*

## Задание 2

Для сравнения метрики конверсии в покупку между двумя группами с размером выборки по 10 000 элементов в каждой группе лучше всего использовать **z-тест**, так как объём выборки достаточно большой (больше 30), что позволяет использовать нормальное распределение для вычисления статистической значимости.
**T-тест** Стьюдента также может использоваться в данном случае, но z-тест является более точным и надёжным при больших размерах выборки.